lib/reline/version.rb

```diff
@@ -1,3 +1,4 @@
 module Reline
   VERSION = '0.3.8'
+  COMPATIBLE_READLINE_VERSION = '8.2'
```

Comment:
This is needed for the `version` test. I put the current version of Readline here, but I can't really say whether Reline is truly compatible with this version. Additionally, this will have to be kept up-to-date as necessary in order to properly support the `version` test.